vim-patch:9.1.0276: No pandoc syntax support

clason opened the pull request:
Problem:  No pandoc syntax support
Solution: Add pandoc syntax and compiler plugins
          (Wu, Zhenyu, Konfekt)

closes: vim/vim#14389

https://github.com/vim/vim/commit/7005b7ee7f282b24378c2a844366cb8616cad5d7

Co-authored-by: Wu, Zhenyu <wuzhenyu@ustc.edu>
Co-authored-by: Konfekt <Konfekt@users.noreply.github.com>

clason committed:
vim-patch:9.1.0276: No pandoc syntax support

Problem:  No pandoc syntax support
Solution: Add pandoc syntax and compiler plugins
          (Wu, Zhenyu, Konfekt)

closes: vim/vim#14389

https://github.com/vim/vim/commit/7005b7ee7f282b24378c2a844366cb8616cad5d7

Co-authored-by: Wu, Zhenyu <wuzhenyu@ustc.edu>
Co-authored-by: Konfekt <Konfekt@users.noreply.github.com>

zeertzjq commented:
There is now a separate patch for the `test_compiler.vim` change: https://github.com/vim/vim/commit/d33cb3f65e112bdb2d57139e9a308b1a9c8ffcc2

clason committed:
vim-patch:9.1.0281: CI: fails Test_compiler_completion

Problem:  CI: fails Test_compiler_completion
Solution: Add pandoc compiler

https://github.com/vim/vim/commit/d33cb3f65e112bdb2d57139e9a308b1a9c8ffcc2

Co-authored-by: Christian Brabandt <cb@256bit.org>

zeertzjq commented:
Hmm, the second commit is empty. I moved the `test_compiler.vim` changes to the second commit.

clason commented:
> Hmm, the second commit is empty. I moved the `test_compiler.vim` changes to the second commit.

Yes, that was intentional (I prefer not to merge known broken commits, if I can help it, and forgot the exact commit message format to make `vim-patch` happy. Could just have squash-merged it, but this works too.
